website/docs/spec/index.md

```diff
@@ -522,6 +573,52 @@ A writer may choose to put messages in Chunks to compress record data. This MCAP
 [Footer]
 ```
 
+### Multiple Messages with a Secondary Index
+
+```
+[Header]
+[Secondary Index Key 1]
+[Chunk A]
+  [Schema A]
+  [Channel 1 (A)]
+  [Channel 2 (B)]
+  [Message on 1]
+  [Message on 1]
+  [Message on 2]
+[Message Index 1]
+[Message Index 2]
+[Secondary Message Index 1 (Channel 1)]
+[Secondary Message Index 1 (Channel 2)]
+[Attachment 1]
+[Chunk B]
+  [Schema B]
+  [Channel 3 (B)]
+  [Message on 3]
+  [Message on 1]
+[Message Index 3]
+[Message Index 1]
+[Secondary Message Index 1 (Channel 3)]
+[Secondary Message Index 1 (Channel 1)]
+[Data End]
+[Schema A]
+[Schema B]
+[Channel 1]
+[Channel 2]
+[Channel 3]
+[Secondary Index Key 1]
+[Chunk Index A]
+[Chunk Index B]
+[Secondary Chunk Index 1 (Chunk A)]
+[Secondary Chunk Index 1 (Chunk B)]
+[Attachment Index 1]
+[Statistics]
+[Summary Offset 0x01]
+[Summary Offset 0x05]
+[Summary Offset 0x07]
+[Summary Offset 0x08]
```

Comment:
I think these summary offsets seem wrong, 0x01 is the header and that's not in the summary section.

I think they should be 0x03, 0x04, 0x10, 0x08, 0x12, 0x0A, 0x0B